lib/tapioca/compilers/dsl/base.rb

```diff
@@ -41,6 +43,18 @@ def gather_constants; end
 
         private
 
+        sig { returns(T::Enumerable[Class]) }
+        def all_classes
+          @all_classes = T.let(@all_classes, T.nilable(T::Enumerable[Class]))
+          @all_classes ||= -> { T.cast(ObjectSpace.each_object(Class), T::Enumerable[Class]) }.call
```

Comment:
This is super ugly, but I couldn't find another way around Sorbet complaining about the `T.cast` call and ivar initialization needing `T.let` instead.

Reply:
I changed it to make an extra `.each` at the end of each enumerable (outside the cast) so that Sorbet is not thrown off and we don't have to do any extra work.